python/options.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('file', nargs='+')
+parser.add_argument('--OutputName',action="store",dest="OutputName",default="Myconfig",help="Set Output file name, default is: Myconfig")
```

Comment:
What is the type of OutputName? int, string, boolean, ...? Define the type.

Change the default to be more descriptive.  Also use "camelCase:"

https://en.wikipedia.org/wiki/Camel_case